app/models/concerns/translatable.rb

```diff
@@ -2,7 +2,7 @@ module Translatable
   extend ActiveSupport::Concern
 
   included do
-    has_many :translations, as: :translatable
+    has_many :translations, as: :translatable, autosave: true
```

Comment:
Could you explain the rationale behind making these autosave? Part of the reason I originally chose not to when refactoring translatable was because with the ability to assign these as if they were normal ActiveRecord attributes, they would exhibit the same saving behavior as the rest of the object and other types of associations (i.e. only when using `create` or `update_attributes`, rather than `build`, `assign_attributes` or `new`). Ordinarily, you wouldn't want the line `@instance.label = 'My new label'`, for instance, to persist immediately.